jackson/src/main/java/com/cerner/beadledom/jackson/ObjectMapperProvider.java

```diff
@@ -46,7 +46,7 @@ public ObjectMapper get() {
     // Sets default values before looking at bound features
     objectMapper.setSerializationInclusion(JsonInclude.Include.NON_ABSENT);
     objectMapper.setPropertyNamingStrategy(
-        PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+        PropertyNamingStrategy.SNAKE_CASE);
```

Comment:
Should we make this `LOWER_CAMEL_CASE` for 4.0 and note it in a section about breaking changes and the migration guide? 

I'm not even sure why this is `SNAKE_CASE` ,since we generally add `@JsonProperty("camelCaseName")` with camel case names, unless it's just been this way since way before we even open sourced this and we've kept it for passivity.

Reply:
There may be some old users that are depending on it to be snake case. I'm fine with making a braking change since everything new should be lower camel case.

Reply:
Changed to LOWER_CAMEL_CASE, aec475c4